contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp

```diff
@@ -1139,6 +1138,20 @@ antlrcpp::Any TsqlUnsupportedFeatureHandlerImpl::visitDdl_statement(TSqlParser::
 		return visitChildren(ctx);
 	}
 
+	if (ctx->create_schema())
+	{
+		auto create_schema = ctx->create_schema();
+		if (create_schema->grant_statement().size() > 0)
+		{
+			throw PGErrorWrapperException(ERROR, ERRCODE_FEATURE_NOT_SUPPORTED, "GRANT inside CREATE SCHEMA is not yet supported in Babelfish", getLineAndPos(ctx));
```

Comment:
Nit: Full stop is missing at end of this error message. We are putting full stop in all other cases.

Reply:
done